bindata/network/openshift-sdn-azure/azure-drop-icmp-daemonset-sdn.yaml

```diff
@@ -0,0 +1,116 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: add-iptables
+  namespace: openshift-sdn
+data:
+  add_iptables.sh: |
+   #!/bin/sh
+   echo "Adding ICMP drop rule for '$2' "
+   if iptables -C CHECK_ICMP_SOURCE -p icmp -s $2 -j ICMP_ACTION
+   then
+        echo "iptables already set for $2"
+   else
+        iptables -A CHECK_ICMP_SOURCE -p icmp -s $2 -j ICMP_ACTION
+   fi
+---
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: drop-icmp
+  namespace: openshift-sdn
+  annotations:
+    kubernetes.io/description: |
+      This daemonset configures each node host to drop ICMP packets to its external address.
+    release.openshift.io/version: "{{.ReleaseVersion}}"
+spec:
+  selector:
+    matchLabels:
+      app: drop-icmp
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      # by default, Deployments spin up the new pod before terminating the old one
+      # but we don't want that - because ovsdb holds the lock.
```

Comment:
this comment does not apply to this DaemonSet, which may mean that setting `maxSurge` doesn't apply